sql/src/test/java/io/crate/operation/reference/sys/SysNodesExpressionsTest.java

```diff
@@ -82,14 +81,15 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+@SuppressWarnings("ALL")
```

Comment:
could you suppress only specific warnings, reduced to the minimum possible scope?